config/locales/mfa/en.yml

```diff
@@ -7,6 +7,15 @@ en:
     info: Add another layer of security by selecting a multi-factor authentication
       method. We recommend you select at least (2) two different options in case
       you lose one of your methods.
+    non_restricted:
+      button: Add an authentication method
+      heading: Add another authentication method.
+      info_html: You opted to receive a code by SMS/text or a phone call to sign in
+        after you entered your email and password. You are now required to add
+        another authentication method as a backup in case you lose access to
+        your phone. <a
+        href='https://login.gov/help/get-started/authentication-options'>Learn
+        more</a>
```

Comment:
I'd consider making the URL a variable so we can interpolate it and link to it via a MarketingSite helper which will automatically add the locale segment in: https://github.com/18F/identity-idp/blob/main/app/services/marketing_site.rb#L53-L58

Reply:
Or maybe even break the `learn_more` bit out into a separate translation bit so that we can just do `link_to` in the template and not worry about `_html` here

Reply:
Thanks for noting the marketing site tool.
We use the `_html` all over the place, and with the translation support by the marketing site tool, I think this is sufficient and conforms with our general style.